shared/domains/chelonia/chelonia.js

```diff
@@ -1044,6 +1052,10 @@ export default (sbp('sbp/selectors/register', {
         }
       }
     })
+
+    if (stream) return s
+    // Workaround for <https://bugs.webkit.org/show_bug.cgi?id=215485>
+    return collectEventStream(s)
```

Comment:
Hmm, what about [this comment](https://bugs.webkit.org/show_bug.cgi?id=215485#c4) from that bug?

> I accidentally found out that it only happens if you use the following header:
Cross-Origin-Embedder-Policy `credentialless` instead of `require-corp`.
Other engines are tolerant of this difference.